### DISCUSSION TITLE
Skill decrease

### DISCUSSION THREAD

**Discussion by MirroredGamer** [ppy/osu#13177] [Ideas]
I know how players struggle to go back to their usual skill level after taking a break from the game, I have been a victim of it, but the idea is that depending on how long they haven't been active in the game (or mode, to be specific), the (separate) skill level will decrease as a recommended system to train back to their skill level, and if they are able to overcome the "skill decrease" difficulty, then it will no longer count and will be ignored untill another inactive period.

**Comment by peppy** [comment]
Are you talking about the recommended difficulty?

**Comment by MirroredGamer** [comment]
well, yes, but personally, I have been having a bit of trouble trying to
train myself up because of how the recommended difficulty system works, and
I thought this kind of mechanic might help me and others too

**Comment by iSimpFor** [comment]
maybe we can split recommended difficulty into 2 parts
highest recommended diff and estimated skill diff
where the former is self-explanatory and latter is an estimation of the player's skill after a period of inactivity 
both will show as (estimated skill, highest recommended) to help the player choose what they want to do

**Comment by MirroredGamer** [comment]
This is what I was thinking about
